### PR TITLE
Improve invent section regarding new technology

### PR DESCRIPTION
**Pull request by jansepar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,7 +113,19 @@ our company and culture evolve, so should this document.
 
 ### Invent
 
-#### Don't be afraid to push the boundaries of technology, but [be conservative with your tech stack](http://mcfunley.com/choose-boring-technology).
+#### New problems don't require new technology
+
+When building a software that solves a problem, it's unlikely that you need new technology to it.
```
Comment (jeremywiebe) [inline README.md]: When building ~~a~~ software ...

File: README.md [hunk 2]
```diff
@@ -115,7 +113,19 @@ our company and culture evolve, so should this document.
...
+By default, you should [choose boring technology](http://mcfunley.com/choose-boring-technology).
+
+To make such technology decisions easy, it's important to maintain a set of best practices outlining what technology
+to use when solving particular classes of problems.
+
+Mobify best practices can be found [here](https://mobify.atlassian.net/wiki/display/PLAT/Coding+and+Design+Best+Practices).
+
+
+Don't be afraid to push the boundaries of technology, but [be conservative with your tech stack](http://mcfunley.com/choose-boring-technology).
 - What this means is that the things you are building should be pushing boundaries, not the things you are building them with. [Here is a list of technology decisions we have evaluated in the past](https://mobify.atlassian.net/wiki/questions/77168927/what-should-i-consider-when-evaluating-a-new-service-feature-or-technology-what-criteria-do-we-use-in-our-decisions).
```
Comment (jeremywiebe) [inline README.md]: This sentence now seems redundant given the changes above.

**Comment by ericmuyser** [comment]
Glanced over it, looks good to me!

File: README.md [hunk 3]
```diff
@@ -115,7 +113,19 @@ our company and culture evolve, so should this document.
...
+#### New problems don't require new technology
+
+When building a software that solves a problem, it's unlikely that you need new technology to it.
+The things you are building should be pushing boundaries, not the things you are building them with.
+By default, you should [choose boring technology](http://mcfunley.com/choose-boring-technology).
+
+To make such technology decisions easy, it's important to maintain a set of best practices outlining what technology
+to use when solving particular classes of problems.
+
+Mobify best practices can be found [here](https://mobify.atlassian.net/wiki/display/PLAT/Coding+and+Design+Best+Practices).
```
Comment (mikenikles) [inline README.md]: Keep in mind that this repo is public, but the wiki is not.

Comment (jansepar) [inline README.md]: Yep - we try to be general when possible but this is also a tool we want Mobifyers use, and sometimes it's helpful to point to internal things. But when doing so, we should make it explicit that it's for Mobifyers only. (IMO anyways!)

File: README.md (hunk 2 above)
Comment (jansepar) [inline README.md]: Whoops just forgot to remove it.

**Comment by jansepar** [comment]
@jeremywiebe updated based on your feedback!

**Comment by jansepar** [comment]
👍 from @jeremywiebe!

**Pull request merged**